templates/html/GuideRenderer.php

```diff
@@ -164,20 +163,8 @@ public function getGuideReferences()
      */
     protected function fixMarkdownLinks($content)
     {
-        $doc = new DOMDocument();
-        $doc->loadHTML($content);
-
-        foreach ($doc->getElementsByTagName('a') as $link) {
-            $href = $link->getAttribute('href');
-            if (strpos($href, '.md') === false) {
-                continue;
-            }
-
-            $href = $this->guidePrefix . str_replace('.md', '.html', $href);
-            $link->setAttribute('href', $href);
-        }
-
-        return $doc->saveHTML();
+        $content = preg_replace('/href\s*=\s*"([^"\/]+)\.md(#.*)?"/i', 'href="' . $this->guidePrefix . '\1.html\2"', $content);
+        return $content;
```

Comment:
```suggestion
        return preg_replace('/href\s*=\s*"([^"\/]+)\.md(#.*)?"/i', 'href="' . $this->guidePrefix . '\1.html\2"', $content);
```

Reply:
Seems irrelevant to current PR. Probably relates to #226, but it's already merged.